axe-linter.sh

```diff
@@ -1,7 +1,19 @@
-#! /bin/bash 
+#! /bin/bash
+
+set -xe
+
+throw() {
+  echo "Error: $*"
```

Comment:
We should send error messages to stderr.

```suggestion
  echo "Error: $*">&2
```